kratos/tests/cpp_tests/sparse_matrices/test_sparse_graph_serial.cpp

```diff
@@ -201,8 +204,8 @@ bool CheckGraph(
 }
 
 bool CheckCSRGraphArrays(
-    const vector<SparseTestingInternals::IndexType>& rRowIndices,
-    const vector<SparseTestingInternals::IndexType>& rColIndices,
+    const span<SparseTestingInternals::IndexType>& rRowIndices,
+    const span<SparseTestingInternals::IndexType>& rColIndices,
```

Comment:
how about wrapping this in `Kratos::span` like e.g. `shared_ptr`
=> this will make the transition to C++20 `std::span` much easier

Reply:
anyhow, i'll add the Kratos:: everywhere (it is quickly done)